app/services/clients/connector.client.js

```diff
@@ -97,6 +23,15 @@ function ConnectorClient (connectorUrl) {
 }
 
 ConnectorClient.prototype = {
+  getAccountByServiceIdAndAccountType: async function (params) {
+    const url = `${this.connectorUrl}/v1/api/service/{serviceId}/account/{accountType}`
+      .replace('{serviceId}', params.serviceId)
+      .replace('{accountId}', params.accountType)
```

Comment:
The replaces here should be wrapped in `encodeURIComponent` to escape special charactors.

Reply:
I'll do this